control.lua

```diff
@@ -205,103 +328,46 @@ end
 script.on_event(defines.events.on_tick, tick)
 
 
--- Update list of stations (because players can change station names)
-function update_list_stations()
```

Comment:
I do not remember why I did not use the event `on_entity_renamed` when I wrote this function (but I must have had a good reason at the time). Anyway, if it works with the event, it is ok with me.

Reply:
They are adding/fixing API features all the time so there are lots of mods with funny workarounds for things that are easier in the latest version.  I first thought to do this because I noticed the CPU time for Dispatcher pulsating as it redid the station list periodically.  I like the new code because it uses no CPU at all if you do not change your rail network stations.

Reply:
> Since 0.18 is out, and you made the changes required for the mod to work with it, I guess you did not plan to make a version for 0.17 (this is fine with me)?

I would like to release one version as 0.17 before moving on to 0.18.  It will make one of my saves better behaved.  Only the changes to signals.lua and items.lua are related to 0.18.

Reply:
Yes, it is much better for CPU, more readable, etc. So keep it that way.